packages/code-du-travail-frontend/src/outils/IndemniteLicenciement/Wizard.js

```diff
@@ -9,14 +9,14 @@ import { StepItems } from "./StepItems";
 import { PrevNextBar } from "./PrevNextBar";
 
 function Wizard({
-  initialValues = {},
+  initialSteps,
   initialStepIndex = 0,
-  steps,
-  onSubmit,
-  onUpdate,
-  rules = null
+  initialValues = {},
+  Rules = null,
+  stepReducer
 }) {
   const [stepIndex, setStepIndex] = useState(initialStepIndex);
```

Comment:
Can we merge this state part into the reducer ?

Reply:
I don't think so because the stepIndex is something internal to the wizard. So asking to index.js to setup that it the reducer would mean it could also impact the way pagination is handled. But it should not except for the initial step index. The only thing index can configure is the steps displayed

Reply:
in fact I would leave the stepReducer up to the user but include it in a "WizzardReducer" wich handle the setIndex something like
```js
case steps: return {...state, steps: stepReducer(state.steps, action)}
```
since reducers can be combined.

Reply:
Ok I added this to the issue aswell, I will get back at it as soon as the planning gets more lightweight.